assets/airports/airportLoadList.js

```diff
@@ -200,13 +200,13 @@ window.AIRPORT_LOAD_LIST = (function() { // eslint-disable-line wrap-iife
             premium: false,
             wip: false
         },
-        // {
-        //     icao: 'kdca',
-        //     level: 'medium',
-        //     name: 'Reagan National Airport',
-        //     premium: false,
-        //     wip: true
-        // },
+        {
+            icao: 'kdca',
+            level: 'medium',
```

Comment:
34acph so level should be easy. If you want you can increase the traffic a bit to make it medium. 40acph would still be realistic for DCA

Reply:
As a single-runway airport, it can be tough to work that much traffic. Realistically, they work about 35-45 an hour continuously for about 10 hours solid (more or less). It's a pretty crazy operation over there. But regardless, I would recommend a hard maximum of 40acph; even that will be very, very challenging to maintain the appropriate runway separation without delaying arrivals, etc.